web/package.json

```diff
@@ -106,6 +106,7 @@
     "@perses-dev/dashboards": "0.51.0-rc.0",
     "@perses-dev/plugin-system": "0.51.0-rc.0",
     "@tanstack/react-query": "4.36.1",
+    "cheerio":"1.0.0-rc.12",
```

Comment:
This was required to fix https://github.com/cheeriojs/cheerio/issues/4024 when regenerating the locales with `npm run i18n`.